v2.1/reference/sql/functions-and-operators/expressions-pushed-down.md

```diff
@@ -0,0 +1,141 @@
+| id                  | count    | task | operator info                                              |
++---------------------+----------+------+------------------------------------------------------------+
+| TableReader_7       | 0.00     | root | data:Selection_6                                           |
+| └─Selection_6       | 0.00     | cop  | gt(test.t.a, 2), lt(test.t.a, 2)                           |
+|   └─TableScan_5     | 10000.00 | cop  | table:t, range:[-inf,+inf], keep order:false, stats:pseudo |
++---------------------+----------+------+------------------------------------------------------------+
+3 rows in set (0.00 sec)
+
+tidb> insert into mysql.expr_pushdown_blacklist values('<'), ('>');
+Query OK, 2 rows affected (0.00 sec)
+Records: 2  Duplicates: 0  Warnings: 0
+
+tidb> admin reload expr_pushdown_blacklist;
+Query OK, 0 rows affected (0.00 sec)
+
+tidb> explain select * from t where a < 2 and a > 2;
++---------------------+----------+------+------------------------------------------------------------+
+| id                  | count    | task | operator info                                              |
++---------------------+----------+------+------------------------------------------------------------+
+| Selection_5         | 8000.00  | root | gt(test.t.a, 2), lt(test.t.a, 2)                           |
+| └─TableReader_7     | 10000.00 | root | data:TableScan_6                                           |
+|   └─TableScan_6     | 10000.00 | cop  | table:t, range:[-inf,+inf], keep order:false, stats:pseudo |
++---------------------+----------+------+------------------------------------------------------------+
+3 rows in set (0.00 sec)
+
+tidb> delete from mysql.expr_pushdown_blacklist where name = '>';
+Query OK, 1 row affected (0.00 sec)
+
+tidb> admin reload expr_pushdown_blacklist;
+Query OK, 0 rows affected (0.00 sec)
+
+tidb> explain select * from t where a < 2 and a > 2;
++-----------------------+----------+------+------------------------------------------------------------+
+| id                    | count    | task | operator info                                              |
++-----------------------+----------+------+------------------------------------------------------------+
+| Selection_5           | 2666.67  | root | lt(test.t.a, 2)                                            |
+| └─TableReader_8       | 3333.33  | root | data:Selection_7                                           |
+|   └─Selection_7       | 3333.33  | cop  | gt(test.t.a, 2)                                            |
+|     └─TableScan_6     | 10000.00 | cop  | table:t, range:[-inf,+inf], keep order:false, stats:pseudo |
++-----------------------+----------+------+------------------------------------------------------------+
+4 rows in set (0.00 sec)
+```
+
+注：
+
+1. `admin reload expr_pushdown_blacklist` 只对执行该 SQL 的 TiDB server 生效，若需要集群中所有 TiDB server 生效，需要在每台 TiDB server 上执行该 SQL。
+2. 表达式黑名单功能在 v3.0.0 及以上版本中支持。
```

Comment:
2.1 do not support the expression blacklist.
So we do not need to add the related part in 2.1.

Reply:
ok, I will delete it.